README.md

```diff
@@ -9,6 +9,49 @@ two components, namely import and export adapter.
 The Import Adapter converts incoming DIMSE requests to corresponding DICOMWeb requests and passes the converted results back to the DIMSE client. The following requests are supported:
 - C-STORE to STOW-RS
 - C-FIND to QIDO-RS
+- C-MOVE uses QIDO-RS to determine which instances to transfer, then for each instance executes a 
+WADO-RS request to fetch the instance and a C-STORE request to transfer it to the C-MOVE destination 
+
+Available AET destinations for C-MOVE are configured via an AET dictionary json file, 
+which can be specified either by using the "--aet_dictionary" command line parameter or 
+specifying the "ENV_AETS_JSON" environment variable.
+
+Here is an example JSON dictionary:
+```shell
+[
+	{
+		"name": "WEASIS", 
+		"host": "localhost", 
+		"port": 11113
+	},
+	{
+		"name": "AESKULAP", 
+		"host": "192.168.0.1", 
+		"port": 11114
+	},
+	{
+		"name": "GINKGO_001",
+		"host": "some.domain.name.com",
+		"port": 6666
+	},
+	...
+]
+```
+
+```shell
```

Comment:
Can we move this to below "The following configuration needs to be added to the dicom-adapter.yaml file to use CMOVE. 
Please see the Deployment using Kubernetes section for more information." line?

Reply:
Ok